lib/libutee/include/user_ta_header.h

```diff
@@ -115,9 +115,6 @@ extern struct __elf_phdr_info __elf_phdr_info;
 #define TA_PROP_STR_STACK_SIZE		"gpd.ta.stackSize"
 #define TA_PROP_STR_VERSION		"gpd.ta.version"
 #define TA_PROP_STR_DESCRIPTION		"gpd.ta.description"
-#define TA_PROP_STR_UNSAFE_PARAM	"op-tee.unsafe_param"
```

Comment:
I think the idea here was to report to the TA some of its flags as properties. It seems that it wasn't completely implemented though.

Reply:
@etienne-lms, what do you think? Do we care about these?

Reply:
I would say no, we don't care.
I even didn't remember we had these macros.
Maybe we should get rid of all of these TA_PROP_STR_*?

Reply:
> I would say no, we don't care.

OK, agree.

> I even didn't remember we had these macros.

Me too! :-)

> Maybe we should get rid of all of these TA_PROP_STR_*?

No, the other are used and required by GP.

Reply:
> > Maybe we should get rid of all of these TA_PROP_STR_*?
>
> No, the other are used and required by GP.

Yes, sorry, I missed occurrences in **ta/arch/arm/user_ta_header.c**.